docs/guide/driver-queue-interop.md

```diff
@@ -0,0 +1,51 @@
+Queue Interop Driver (php-enqueue)
+==================================
+
+The driver works with many queue brokers.
+
+Full list of supported brokers you can find on the [Queue Interop](https://github.com/queue-interop/queue-interop) page
+
+To get it works you have to install and setup one of the supported implementation.
```

Comment:
In order for it to work you need to install and configure one the implementations supported.